<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>732</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: rgb(20, 20, 20); /* 원하는 색상으로 변경 */
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,22" columnstretch="1,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
          </property>
          <property name="text">
           <string>Default</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
          </property>
          <property name="text">
           <string>Experiment</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="non_history_txt">
          <property name="styleSheet">
           <string notr="true">QTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
}
QTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhMultiLine</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextEdit" name="history_txt">
          <property name="styleSheet">
           <string notr="true">QTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
}
QTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhMultiLine</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
        <item>
         <widget class="QPlainTextEdit" name="input_text">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPlainTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
}
QPlainTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Enter your message...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,3,1" columnstretch="1,8,1">
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="send_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(184, 247, 185);
    border: 1px solid rgb(184, 247, 185); /* 원하는 RGB 색상으로 변경 */
    border-radius: 5px;                  /* 모서리를 둥글게 (선택 사항) */
}
QPushButton:hover {
	color: rgb(184, 247, 185);
    background-color: rgb(50, 120, 50); /* 마우스 올렸을 때 더 진한 색 */
    border: 1px solid rgb(184, 247, 185); /* 테두리도 함께 진하게 (선택 사항) */
}

QPushButton:pressed {
	color: rgb(184, 247, 185);
    background-color: rgb(100, 180, 100); /* 버튼 눌렀을 때 가장 진한 색 */
    border: 1px solid rgb(184, 247, 185); /* 테두리도 함께 진하게 (선택 사항) */
}</string>
            </property>
            <property name="text">
             <string>send</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Setting">
       <attribute name="title">
        <string>setting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,6,1,2,10">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
          </property>
          <property name="text">
           <string>Api-key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="api_key_txt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
}
QLineEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="placeholderText">
           <string>Enter your api key...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
          </property>
          <property name="text">
           <string>Prompt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="prompt_txt">
          <property name="styleSheet">
           <string notr="true">QTextEdit {
	color: white;
    background-color: rgb(40, 40, 40); /* 원하는 색상으로 변경 */
	border: 1px solid rgb(255,255,255);
}
QTextEdit:focus {
    border: 1px solid rgb(184, 247, 185);
    outline: none;           /* 기본 포커스 아웃라인 제거 */
}</string>
          </property>
          <property name="placeholderText">
           <string>Enter your prompt...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(166, 217, 171);
}</string>
          </property>
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="temp_combo">
          <property name="styleSheet">
           <string notr="true">QComboBox {
	color: white;
	background-color: rgb(40,40,40);
    border: 1px solid white; /* 2px 두께의 흰색 실선 테두리 */
    border-radius: 3px;      /* (선택 사항) 모서리를 둥글게 */
    padding: 1px 18px 1px 3px; /* 텍스트와 드롭다운 버튼을 위한 패딩 */
    min-width: 6em;          /* (선택 사항) 최소 너비 설정 */
}
QComboBox QAbstractItemView {
    color: white; /* 항목 텍스트 색상을 흰색으로 설정 */
    background-color: #333333; /* 드롭다운 목록 배경색 (잘 보이도록 어둡게) */
    selection-background-color: #558855; /* 항목 선택 시 배경색 */
    selection-color: white; /* 선택된 항목의 글씨색 (흰색으로 유지) */
    border: 1px solid white; /* 드롭다운 목록 테두리 (선택 사항) */
}</string>
          </property>
          <item>
           <property name="text">
            <string>select temperature...</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.0</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
   </widget>
   <addaction name="menu_Setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>send_btn</tabstop>
  <tabstop>input_text</tabstop>
  <tabstop>non_history_txt</tabstop>
  <tabstop>history_txt</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>temp_combo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>send_btn</sender>
   <signal>clicked()</signal>
   <receiver>input_text</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
